.circleci/config.yml

```diff
@@ -53,9 +53,21 @@ jobs:
       - install_rust
       - install_node_devlibs
       - print_current_versions
+      - run:
```

Comment:
🔧 Instead of having this step in here, we probably should add it to `Install node and development libraries` step.
Because, if I understand it correctly, we will now use a different ssl version for the release-build and for our debug-test build.

Reply:
yes correct, it uses a different openssl version for release-build and debug-build.

But then, we all use different versions locally anyway (mac does not use openssl but libressl).

Note that we run all e2e with the binary release in any case so we do validate that before releasing.

If we were to static build for debug-test, we would need to compile openssl for each CI build, which will significantly increase the build time by 3min30s.
I think this build time increase is not worth it.

Reply:
Fair, thanks for the clarification.